build-system/tasks/extension-helpers.js

```diff
@@ -602,10 +588,16 @@ async function buildExtensionCss(extDir, name, version, options) {
   if (Array.isArray(options.cssBinaries)) {
     parallel.push(
       ...options.cssBinaries.map((name) => {
-        return jsifyCssAsync(`${extDir}/${name}.css`).then((css) => {
+        return jsifyCssAsync(`${extDir}/${name}.css`).then(async (css) => {
           writeCssBinaries(`${name}-${version}.css`, css);
+          if (options.bento) {
+            await buildBentoCss(name, version, css);
+          }
           if (isAliased) {
             writeCssBinaries(`${name}-${aliasBundle.aliasedVersion}.css`, css);
+            if (options.bento) {
+              await buildBentoCss(name, aliasBundle.aliasedVersion, css);
```

Comment:
This is getting hairy. The Bento aliased version is not output correctly since it doesn't write to `dist/v0/`, and the Bento rename is being done twice. I think it's as good a time as any to refactor: remove the specific `buildBentoCss`, and change this function:

```js
async function buildExtensionCss(extDir, name, version, options) {
  const aliasBundle = extensionAliasBundles[name];
  const aliasVersion =
    aliasBundle?.version == version ? aliasBundle.version : null;

  const versions = [version, aliasVersion].filter(Boolean);
  const bundles = [name, ...(options.cssBinaries || [])];

  const parallel = bundles.map(async (name) => {
    const css = await jsifyCssAsync(`${extDir}/${name}.css`);
    await writeCssBinaries(name, versions, css);

    if (options.bento) {
      // Build bento-*.css using the compiled amp-// result as source.
      // It replaces all selectors for elements <amp-*> with <bento-*>.
      // As a result of taking already minified code as source, this function is
      // fairly fast and not cached.
      const renamedCss = await renameSelectorsToBentoTagNames(css);
      const bentoName = getBentoName(name);
      await writeCssBinaries(bentoName, versions, renamedCss);
      await writeVersions(`dist/v0/${bentoName}`, 'css', versions, renamedCss);
    }
  });

  await Promise.all(parallel);
}
```

`writeCssBinaries` runs `writeVersions`:

```js
/**
 * @param {string} name
 * @param {string[]} versions
 * @param {string} css
 * @return {!Promise}
 */
async function writeCssBinaries(name, versions, css) {
  const jsCss = 'export const CSS = ' + JSON.stringify(css) + ';\n';
  await writeVersions(`build/${name}`, 'js', versions, jsCss);
  await writeVersions(`build/${name}`, 'css', versions, css);
}

/**
 * @param {string} prefix
 * @param {string} fileExtension
 * @param {string[]} versions
 * @param {string} content
 * @return {!Promise<void>}
 */
async function writeVersions(prefix, fileExtension, versions, content) {
  for (const version of versions) {
    const outfile = `${prefix}-${version}.${fileExtension}`;
    await fs.outputFile(outfile, content);
  }
}
```